docs/tutorial/examples/clip.py

```diff
@@ -0,0 +1,14 @@
+from onnxscript import opset15 as op
+from onnxscript import script
+
+
+@script()
+def Clip(input, min, max):
+    # The inputs min and max are optional inputs
+    result = input
+    # OptionalHasElement is used to check if an optional input is not None
+    if op.OptionalHasElement(min):
+        result = op.Where(result < min, min, result)
```

Comment:
Does it mean we don’t need to use optional get element?

Reply:
I do think we need to use OptionalGetElement. Otherwise, the checker will likely complain.